src/mergeCoverage.js

```diff
@@ -34,22 +35,39 @@ function generateSummaryReport(dir, coverageMap) {
   reports.create("json-summary").execute(context);
 }
 
-exports.mergeCoverageAndGenerateSummaryReport =
+function generateHtmlReport(config, coverageMap) {
+  const dir = config.input.coverageHtmlPath;
```

Comment:
I think this always needs to be the input path (to `config.input.coverageSummaryPath`) - I'm not sure how you influence the output path of the report (maybe more parameters to `create`?)